modules/metastorage-client/src/main/java/org/apache/ignite/internal/metastorage/client/CursorImpl.java

```diff
@@ -96,4 +79,42 @@
             throw new IgniteInternalException(e);
         }
     }
+
+    @Override
+    public boolean hasNext() {
```

Comment:
javadoc is missing for hasNext() and next()

Reply:
Thx! Fixed.